src/EVM/Expr.hs

```diff
@@ -651,8 +641,8 @@ simplify e = if (mapExpr go e == e)
     go (CopySlice (Lit 0x0) (Lit 0x0) (Lit 0x0) _ dst) = dst
 
```

Comment:
Maybe we could also add a simplification of `(WAddr (LitAddr val))` to `(Lit $ into val)`?

Reply:
done.